src/sst/core/impl/timevortex/timeVortexPQ.cc

```diff
@@ -162,19 +165,20 @@ class TimeVortexPQ_ts : public TimeVortexPQBase<true>
 {
 public:
     SST_ELI_REGISTER_DERIVED(
-        TimeVortex,
-        TimeVortexPQ_ts,
-        "sst",
-        "timevortex.priority_queue.ts",
-        SST_ELI_ELEMENT_VERSION(1,0,0),
-        "Thread safe verion of TimeVortex based on std::priority_queue.  Do not reference this element directly, just specify sst.timevortex.priority_queue and this version will be selected when it is needed based on other parameters.")
+        TimeVortex, TimeVortexPQ_ts, "sst", "timevortex.priority_queue.ts", SST_ELI_ELEMENT_VERSION(1, 0, 0),
```

Comment:
Convention is to have one item per line.  This should remain as it currently is.

Reply:
It was a `clang-format` difference. Newer versions apparently do that.